### ISSUE TITLE
URL with # character flow problem

### ISSUE THREAD

**Issue by ssmali1505** [a85/POSTMan-Chrome-Extension#9203]
Hi Team 

I am facing one problem in restapi calling as follows,
I have two endpoints 
1. To get all :   http://<domain name>:<port>/users/ 
2. To get single :   http://<domain name>:<port>/users/<id>

So QA is trying to hit following 2 urls
1. http://<domain name>:<port>/users/$12345
2.  http://<domain name>:<port>/users/#12345

So in 1st url hit backend is responding correctly as "$12345" is not found.
but in 2nd case backend is responding with get all api.

So, in short, we have observed like if # is present in url then postman ignore the next URL part and sending previous
url only to the server and that what is it matching with get all api and we get the result.

Please confirm it?

![postman](https://user-images.githubusercontent.com/3482596/97418519-e9678e00-192e-11eb-8c33-bc6c149d68f8.png)

**Comment by codenirvana** [comment]
@ssmali1505 You have to encode `#` to `%23` to send the URL [fragment](https://en.wikipedia.org/wiki/URI_fragment).

Closing as duplicate of https://github.com/postmanlabs/postman-app-support/issues/6023
